cost/kubecost/cluster/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v0.3.2
+
+- Replace usage of Kubecost deprecated endpoints for newest versions
```

Comment:
This is awkwardly worded. Recommended change:

- Updated API endpoints for Kubecost. Functionality unchanged.